src/client/components/MarkdownInputModal/MarkdownInputModal.less

```diff
@@ -0,0 +1,15 @@
+.fullscreen {
```

Comment:
`fullscreen` is a too common name for CSS class in this case. It can may cause to style conflicts in future. Make this class more local-scoped please.